app/views/layouts/_segment_analytics.html.erb

```diff
@@ -0,0 +1,19 @@
+<!-- Global site tags for Segment analytics-->
+<script>
+  const SEGMENT_IDS = {
+    // TODO: Replace these with new accounts after they get set up
+    // These are designed to be public: https://community.segment.com/t/m26sng/writekey-accessible-by-anyone
+    prod: "7gUE8JMXEkhEo9ioD1uHke1RgpDFxxlG",
```

Comment:
Do you want to load this from server as discussed offline?

Reply:
Just noticed the comment above.

Reply:
👍